Testscripts/Linux/VERIFY-LINUX-CONFIGURATION.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+from azuremodules import *
+import os
+import os.path
+
+root_bash_hist_file = '/root/.bash_history'
+root_bash_hist_file_default = '/root/default_bash_history'
+
+root_password_verify_result = False
+sshd_config_check_result = False
+last_console_check_result = False
+bash_history_verify_result = False
+
+
+'''
+sshdFilePath = "/etc/ssh/sshd_config"
+expectedString = "ClientAliveInterval"
+commentedLine = "#ClientAliveInterval"
+'''
+
+
+def IsBashHistFileEmpty(file):
+    if os.stat(file).st_size == 0:
+        return True, ''
+    else:
+        with open(file, 'r') as f:
+            return False, f.read()
+
+
+def VerifySSHDConfig():
+    global sshd_config_check_result
+    RunLog.info("Checking ClientAliveInterval is into the /etc/ssh/sshd_config file")
+    ClientAliveIntervalLines = Run("cat /etc/ssh/sshd_config | grep -i '^ClientAliveInterval' | wc -l | tr -d ' ' | tr -d '\n'")
+    CommentClientAliveIntervalLines = Run("cat /etc/ssh/sshd_config | grep -i '^#ClientAliveInterval' | wc -l | tr -d ' ' | tr -d '\n'")
+
+    if (int(CommentClientAliveIntervalLines) != 0):
+        print ("CLIENT_ALIVE_INTERVAL_COMMENTED")
+        RunLog.info("Commented ClientAliveInterval found in /etc/ssh/sshd_config and continue to check the expected interval.")
+
+    if (int(ClientAliveIntervalLines) != 1 ):
+        print ("CLIENT_ALIVE_INTERVAL_FAIL")
+        RunLog.error('ClientAliveInterval is not into the /etc/ssh/sshd_config file.')
+    else:
+        RunLog.info("ClientAliveInterval is into in /etc/ssh/sshd_config.")
+        RunLog.info("Checking the interval.")
+        ClientAliveIntervalValue = Run("cat /etc/ssh/sshd_config | grep -i '^ClientAliveInterval' | awk '{print $2}'")
+        if (int(ClientAliveIntervalValue) < 181 and int(ClientAliveIntervalValue) > 0):
+            print ("CLIENT_ALIVE_INTERVAL_SUCCESS")
+            RunLog.info("ClientAliveInterval " + 'is ' + ClientAliveIntervalValue)
+            sshd_config_check_result = True
+        else:
+            print ("CLIENT_ALIVE_INTERVAL_FAIL")
+            RunLog.error("ClientAliveInterval is " + ClientAliveIntervalValue + " is not expected.")
+
+
+def VerifyRootPassword():
+    global root_password_verify_result
+    RunLog.info("Checking if root password is deleted or not...")
```

Comment:
if root password is expected to be deleted, please change the message to below, to make it clear on what is the expected behavior.
"Checking if root password is deleted"